schemas/firefox-accounts/account-ecosystem/account-ecosystem.1.schema.json

```diff
@@ -12,6 +12,21 @@
       ]
     }
   ],
+  "mozPipelineMetadata": {
+    "bq_dataset_family": "firefox_accounts",
+    "bq_table": "account_ecosystem_v1",
+    "jwe_mappings": [
+      {
+        "decrypted_field_path": "ecosystem_user_id",
+        "source_field_path": "ecosystem_anon_id"
+      },
+      {
+        "decrypted_field_path": "previous_ecosystem_user_ids",
+        "source_field_path": "previous_ecosystem_anon_ids"
+      }
+    ],
+    "uri_scheme": "structured"
```

Comment:
I'm a bit hung up on uri_scheme since semantically its a property of the input but this metadata as spec'd will only be used on output. For structured aet in particular I'd expect a new input uri scheme of `stackdriver`.